python_modules/dagster/dagster/_core/definitions/auto_materialize_policy.py

```diff
@@ -56,9 +56,7 @@ def eager(
             on_missing=True,
             on_upstream_data_newer=True,
             for_freshness=True,
-            time_window_partition_scope_minutes=int(
-                time_window_partition_scope.total_seconds() / 60
-            )
+            time_window_partition_scope_minutes=time_window_partition_scope.total_seconds() / 60
```

Comment:
as an int, the default `datetime.timedelta.resolution` gets floored to 0 so we'd need another approach for selecting the 1 latest. FreshnessPolicies take floats as well